lib/improver/wind_downscaling.py

```diff
@@ -109,7 +109,7 @@ def process(self):
                 A 2D array of friction velocities
 
         """
-        ustar = np.full(self.u_href.shape, RMDI)
+        ustar = np.full(self.u_href.shape, RMDI, dtype=np.float32)
```

Comment:
This seems to be a suitable approach for now. Based on the discussion inhttps://stackoverflow.com/questions/5350342/can-i-set-float128-as-the-standard-float-array-in-numpy and https://stackoverflow.com/questions/5721831/python-making-numpy-default-to-float32, there doesn't seem to be a way to set the default dtype for numpy arrays more globally, so we might need to explore writing our own wrapper functions later.